playwright/navigation.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { expect, test } from '@playwright/test';
+
+test('check content', async ({ page }) => {
+  await page.goto('/');
+  await expect(page.getByTestId('myHero')).toBeVisible();
+  await expect(page.getByTestId('whoAmI')).toBeVisible();
+  await expect(page.getByTestId('myWork')).toBeVisible();
+  await expect(page.getByTestId('footer')).toBeVisible();
```

Comment:
let's also test that clicking on all of the buttons in the hero section at the top also scrolls down the page

Reply:
Added an assertion after the click
[3a053c8](https://github.com/upleveled/portfolio-cms/pull/112/commits/3a053c8819508d63f642d461075b68d0ad644cef)